core/src/main/java/org/ldaptive/extended/SyncInfoMessage.java

```diff
@@ -278,7 +282,7 @@ public void addEntryUuids(final UUID... uuids)
    */
   public void setEntryUuids(final Set<UUID> uuids)
   {
-    entryUuids.addAll(uuids);
+    entryUuids = uuids;
```

Comment:
Just noting that copying the input is a little safer to avoid the caller pulling the rug out from under you with a shared reference.

Reply:
I'm not sure why this method exists. It's not used internally. Comparison with other parsing classes that manage collections indicate that `addFoo` is generally all that is needed.

I've deprecated this method for removal in the next minor release.